2022-08-01 19:02:10,321 INFO    Thread-27 :12723 [wandb_setup.py:_flush():76] Configure stats pid to 12723
2022-08-01 19:02:10,321 INFO    Thread-27 :12723 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 19:02:10,321 INFO    Thread-27 :12723 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 19:02:10,321 INFO    Thread-27 :12723 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_a', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': 'wi15u5ki', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-7d9wmz9t/config-wi15u5ki.yaml', 'sweep_id': '7d9wmz9t'}
2022-08-01 19:02:10,321 WARNING Thread-27 :12723 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 19:02:10,321 INFO    Thread-27 :12723 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 19:02:10,322 INFO    Thread-27 :12723 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_190210-wi15u5ki/logs/debug.log
2022-08-01 19:02:10,322 INFO    Thread-27 :12723 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_190210-wi15u5ki/logs/debug-internal.log
2022-08-01 19:02:10,325 INFO    Thread-27 :12723 [wandb_init.py:init():495] calling init triggers
2022-08-01 19:02:10,326 INFO    Thread-27 :12723 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 64, 'IMS_PER_BATCH': 4, 'iteration': 1000, 'learning_rate': 0.0006803592008525793, 'model': 'COCO-Detection/faster_rcnn_R_101_C4_3x.yaml'}
config: {}
2022-08-01 19:02:10,329 INFO    Thread-27 :12723 [wandb_init.py:init():548] starting backend
2022-08-01 19:02:10,329 INFO    Thread-27 :12723 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 19:02:10,339 INFO    Thread-27 :12723 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 19:02:10,362 INFO    Thread-27 :12723 [backend.py:ensure_launched():222] started backend process with pid: 25695
2022-08-01 19:02:10,363 INFO    Thread-27 :12723 [wandb_init.py:init():558] backend started and connected
2022-08-01 19:02:10,377 INFO    Thread-27 :12723 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 64, 'IMS_PER_BATCH': 4, 'iteration': 1000, 'learning_rate': 0.0006803592008525793, 'model': 'COCO-Detection/faster_rcnn_R_101_C4_3x.yaml'}
2022-08-01 19:02:10,381 INFO    Thread-27 :12723 [wandb_init.py:init():634] updated telemetry
2022-08-01 19:02:10,383 INFO    Thread-27 :12723 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 19:02:11,165 INFO    Thread-27 :12723 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 19:02:11,208 INFO    Thread-27 :12723 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 19:02:11,208 INFO    Thread-27 :12723 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 19:02:12,326 INFO    Thread-27 :12723 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 19:02:12,328 INFO    Thread-27 :12723 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 19:02:12,328 INFO    Thread-27 :12723 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 19:02:12,328 ERROR   Thread-27 :12723 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 19:02:12,329 INFO    Thread-27 :12723 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 19:02:13,177 INFO    Thread-27 :12723 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_a/wi15u5ki
2022-08-01 19:02:13,177 INFO    Thread-27 :12723 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-01 19:02:13,178 INFO    Thread-27 :12723 [wandb_run.py:_restore():1834] restore
2022-08-01 19:02:13,183 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:02:13,569 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:02:14,188 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5309
}

2022-08-01 19:02:14,291 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5949
}

2022-08-01 19:02:14,393 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5949
}

2022-08-01 19:02:14,498 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5949
}

2022-08-01 19:02:14,601 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5283
  total_bytes: 5949
}

2022-08-01 19:02:14,703 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}

2022-08-01 19:02:14,805 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}

2022-08-01 19:02:14,907 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}

2022-08-01 19:02:15,010 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}

2022-08-01 19:02:15,112 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}

2022-08-01 19:02:15,899 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}

2022-08-01 19:02:16,517 INFO    Thread-27 :12723 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5949
  total_bytes: 5949
}
local_info {
}

2022-08-01 19:02:17,668 INFO    Thread-27 :12723 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-01 19:02:17,668 INFO    Thread-27 :12723 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-01 19:02:17,670 INFO    Thread-27 :12723 [wandb_run.py:_footer_sync_info():3055] logging synced files
